demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, artifacts_path: str, content_version='', content_bundle_path=
         self.artifacts_path = artifacts_path if artifacts_path else '/home/circleci/project/artifacts'
         self.content_version = content_version
         self.preserve_bundles = preserve_bundles
-        self.only_packs = tools.is_private_repository() or packs
+        self.only_packs = tools.is_external_repository() or packs
```

Comment:
Not sure about the logic here. Before, `only packs` was true if it was a private repo and now it is true if it is an external one?

Reply:
The logic is the same. Just renamed how we call these repos